### PR TITLE
Added implementation for adding courses to the database

### PR DESCRIPTION
**Pull request by OmarHHammad**

### CODE REVIEW

File: TAInformation/views.py [hunk 1]
```diff
@@ -63,34 +63,36 @@ def get(self, request):
         return render(request, "createCourse.html", {})
 
     def post(self, request):
-        noPermissions = canAccess(User.objects.get('role'), 1)
-
+        noPermissions = canAccess(1, 1) #User.objects.get('role')
```
Comment (vaishnavi-manikonda) [inline TAInformation/views.py:66]: I think using User.objects.get('role').role might be better than hardcoding a value in

File: TAInformation/views.py [hunk 2]
```diff
@@ -63,34 +63,36 @@ def get(self, request):
...
+            print(request.POST['name'])
+            newCourse = addCourse(request.POST['name'], request.POST['instructor'],
                                   request.POST['meeting_time'], request.POST['semester'], request.POST['course_type'],
                                   request.POST['description'])
             newCourse.save()
-            request.session["name"] = newCourse.name
-            return redirect("/")
+            request.session["user_id"] = 1
+            return render(request, "dashboard.html",
+                          {"message": "User Created Sucsessfully"})
```
Comment (vaishnavi-manikonda) [inline TAInformation/views.py:79]: *Sucessfully

File: TAInformation/views.py [hunk 3]
```diff
@@ -63,34 +63,36 @@ def get(self, request):
...
         else:
-            newCourse = addCourse(request.POST['name'], request.POST['instructor'], request.POST['lab'],
+            print(request.POST['name'])
+            newCourse = addCourse(request.POST['name'], request.POST['instructor'],
                                   request.POST['meeting_time'], request.POST['semester'], request.POST['course_type'],
                                   request.POST['description'])
             newCourse.save()
-            request.session["name"] = newCourse.name
-            return redirect("/")
+            request.session["user_id"] = 1
```
Comment (vaishnavi-manikonda) [inline TAInformation/views.py:77]: same thing about the hardcoding

File: TAInformation/views.py (hunk 1 above)
Comment (OmarHHammad) [reply TAInformation/views.py]: Fixed

File: TAInformation/views.py (hunk 2 above)
Comment (OmarHHammad) [reply TAInformation/views.py]: Fixed

File: TAInformation/views.py (hunk 3 above)
Comment (OmarHHammad) [reply TAInformation/views.py]: Fixed

**Review by vaishnavi-manikonda — APPROVED** [review]

**Pull request merged**